<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>多目标跟踪分析软件</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>751</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>420</y>
      <width>20</width>
      <height>121</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>585</x>
      <y>0</y>
      <width>211</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>分析模式</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_model"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btn_import">
         <property name="text">
          <string>导入</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_clear">
         <property name="text">
          <string>清空</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_delete">
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListView" name="listView_ImgOrVid">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>显示</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>296</width>
      <height>95</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_Alg">
       <property name="text">
        <string>检测算法</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox_Alg"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btn_start">
       <property name="text">
        <string>开始检测</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>置信阈值</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>IOU阈值</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>工具</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>comboBox_model</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>change_mode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>721</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>879</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_import</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_pushButton_import()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>620</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>522</x>
     <y>-82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listView_ImgOrVid</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_listView_ImgOrVid_doubleClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>677</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>906</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_clear</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_pushButton_clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>689</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>861</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>change_mode()</slot>
  <slot>on_pushButton_import()</slot>
  <slot>on_listView_ImgOrVid_doubleClicked()</slot>
  <slot>on_pushButton_clear()</slot>
 </slots>
</ui>
